R/visualizations.R

```diff
@@ -2,19 +2,16 @@
 #'
 #' Plot a single cell's expression profile against that of a reference sample across all genes. 
 #' 
-#' @param test Numeric matrix of single-cell expression values (usually log-transformed
-#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' @param test A numeric matrix of single-cell expression values where rows are genes and columns are cells.
 #' Alternatively, a \linkS4class{SummarizedExperiment} object containing such a matrix.
-#' @param test.id Integer scalar specifying the index of the target cell to use.
-#' Alternatively, a string containing the name of the cell.
-#' @param ref Numeric matrix of reference dataset expression values (usually log-transformed
-#' or otherwise variance-stabilized), where rows are genes and columns are cells.
+#' @param test.id Integer scalar or character specifying the index or name of the target cell to use.
```

Comment:
"or string". It's rather unfortunate that R calls them character vectors, they should really be referred to as string vectors. If you're coming from C, character vectors suggest character arrays, which are quite different.

Reply:
I have changed all mention of "character" to "string".  Check?